### PR TITLE
Bringing Baba GUIController & UDPReader to the GUI branch (before pulling to main)

### PR DESCRIPTION
**Pull request by Ttiki**
#### This is a:

* [x] bug correction
* [x] new feature
* [ ] documentation enhancement
* [x] code optimization
* [ ] security enhancement
* [x] Other : classes import
* [ ] 
### PR description
Baptiste's work to GUI. The GUI branch will centralize every branch before pulling to main.

### CODE REVIEW

File: GUI/GUIController.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
+package GUI;
```
Comment (Ttiki) [inline GUI/GUIController.java:1]: Bring a better implementation for this system?

File: UDPReader.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
+package network.udp;
```
Comment (Ttiki) [inline UDPReader.java:1]: Finish implementing the UDP Protocol (-> protocol Reader)

**Review by Ttiki — COMMENTED** [review]
This bring new classes as simple .java files. They lack any real implementations to the real code and project.

File: GUI/GUIController.java (hunk 1 above)
Comment (Zaptachaine) [reply GUI/GUIController.java:1]: OUI CHEF

**Review by Zaptachaine — APPROVED** [review]
Impec chef

**Pull request merged**